usercommandmgmt/usercommandmgmt.py

```diff
@@ -0,0 +1,226 @@
+from redbot.core import Config, checks, commands, data_manager
+from .configurable import *
+from .commanddatahandler import *
+import inspect
+import sys
+from .commanddatahandler import Database
+import os
+
+
+# imports customcomm class
+path = os.path.join(
+    os.path.dirname(os.path.dirname(os.path.dirname(inspect.getfile(commands)))),
+    "cogs\customcom",
+)
+sys.path.insert(1, path)
+
+from customcom import (
+    CustomCommands,
+    CommandObj,
+    NotFound,
+    ArgParseError,
+    CommandNotEdited,
+)
+
+
+class Usercommandmgmt(CustomCommands):
+    """My custom cog"""
+
+    def __init__(self, bot):
+        super().__init__(bot)
+        saveFile = os.path.join(
+            data_manager.cog_data_path(cog_instance=self), "user commands.json"
+        )
+        if not os.path.isfile(saveFile):
+            with open(saveFile, "w+") as f:
+                empty = {"db": []}
+                json.dump(empty, f)
+
+        self.activeDb = Database(saveFile)
+
+    # def createDbInstance(self, path):
+    #     if not os.path.isfile(path):
+    #         with open(path, "w+") as f:
+    #             empty = dict()
+    #             json.dump(empty, f)
+
+    #     activeDb = Database(path)
+
+    @commands.command()
+    async def testcomm(self, ctx):
+        """This does stuff!"""
+
+        pass
+
+    @commands.group(aliases=["cc"])
+    @commands.guild_only()
+    async def customcom(self, ctx: commands.Context):
+        """Base command for Custom Commands management."""
+        pass
+
+    @customcom.group(name="create", aliases=["add"], invoke_without_command=True)
+    async def cc_create(self, ctx: commands.Context, command: str.lower, *, text: str):
+        """Create custom commands.
+
+        If a type is not specified, a simple CC will be created.
+        CCs can be enhanced with arguments, see the guide
+        [here](https://docs.discord.red/en/stable/cog_customcom.html).
+        """
+
+        # overrides the mod process and limit check if user is an admin
+        if ctx.message.author.top_role.permissions.administrator:
+            await ctx.send("Admin request detected. Bypassing checks")
+            # custom command is created and the entry is added to the database
+            try:
+                await ctx.invoke(self.cc_create_simple, command=command, text=text)
+                # marks command as created by admin in database, exempted from count
+                await self.activeDb.SaveToDb(command, ctx.message.author.id, True)
+            except:
+                await ctx.send("something went wrong; could not add command")
+                return
+        # normal per-role allowance check and moderation process if user isnt an admin
+        else:
+            # checks if user has any capacity left to make commands based on their allowance
+            if self.EnforceUserCmdLimit(ctx.message.author) == False:
```

Comment:
nit: Here and elsewhere ([PEP8 recommendations](https://www.python.org/dev/peps/pep-0008/#programming-recommendations)), no need for False comparison for a bool:
```suggestion
            if not self.EnforceUserCmdLimit(ctx.message.author):
```